runtime/pom.xml

```diff
@@ -400,6 +400,25 @@
       <artifactId>okhttp-eventsource</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok</artifactId>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
```

Comment:
So its now decided that we are using JPA / Hibernate. Was there some additional discussion  beyond the one we had last week or did it just happen ? Just curious ...